recipes/openapi-spec-validator/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.2.8" %}
+
+package:
+  name: openapi-spec-validator
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/p1c2u/openapi-spec-validator/archive/{{ version }}.tar.gz
+  sha256: 49a0c4dbd89fa66737fd8e93d71071d93c65c8586a4bff89771c65ec897916af
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
This seems like it can be `noarch: python`

Reply:
I did tried with noarch above but the windows build failed with:

`[noarch_python] Noarch package contains binary script: openapi-spec-validator.exe`

Reply:
I'm ok without noarch here. Will merge in a day or so if we don't hear back from someone else on @conda-forge/staged-recipes.

Reply:
That error means you’re missing console script entry points:
https://github.com/p1c2u/openapi-spec-validator/blob/master/setup.py#L69
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Reply:
looks like it worked ! thanks @chrisburr .